fk7263/src/main/java/se/inera/intyg/common/fk7263/model/converter/RespConstants.java

```diff
@@ -0,0 +1,124 @@
+
+    public static final String JOURNALUPPGIFTER_DELSVAR_ID = "6.3";
+    public static final String JOURNALUPPGIFTER_DELSVAR_TEXT = "Journaluppgifter, den";
+
+    public static final String ANNAT_DELSVAR_ID = "6.4";
+    public static final String ANNAT_DELSVAR_TEXT = "Annat, den";
+    public static final String AKTIVITETSBEGRANSNINGAR_CATEGORY_ID = "aktivitetsbegransningar";
+    public static final String AKTIVITETSBEGRANSNINGAR_CATEGORY_TEXT_ID = "fk7263.label.limitation";
+    public static final String AKTIVITETSBEGRANSNINGAR_SVAR_ID = "7";
+
+    public static final String REKOMMENDATIONER_CATEGORY_ID = "rekommendationer";
+    public static final String REKOMMENDATIONER_CATEGORY_TEXT_ID = "fk7263.label.recommendations";
+    public static final String REKOMMENDATIONER_KONTAKT_MED_AF_DELSVAR_ID = "8.1";
+    public static final String REKOMMENDATIONER_KONTAKT_MED_AF_DELSVAR_TEXT_ID = "fk7263.label.recommendations.contact.jobcenter";
+    public static final String REKOMMENDATIONER_KONTAKT_MED_FHV_DELSVAR_ID = "8.2";
+    public static final String REKOMMENDATIONER_KONTAKT_MED_FHV_DELSVAR_TEXT_ID = "fk7263.label.recommendations.contact.healthdepartment";
+    public static final String REKOMMENDATIONER_OVRIGT_DELSVAR_ID = "8.3";
+    public static final String REKOMMENDATIONER_OVRIGT_DELSVAR_TEXT_ID = "fk7263.label.recommendations.contact.other";
+    public static final String BEHANDLING_ELLER_ATGARD_CATEGORY_ID = "behandlingEllerAtgard";
+    public static final String BEHANDLING_ELLER_ATGARD_CATEGORY_TEXT_ID = "fk7263.label.plannedtreatment";
+    public static final String INOM_SJUKVARD_DELSVAR_ID = "9.1";
+    public static final String INOM_SJUKVARD_DELSVAR_TEXT_ID = "fk7263.label.plannedtreatment.healthcare";
+    public static final String ANNAN_ATGARD_DELSVAR_ID = "9.2";
+    public static final String ANNAN_ATGARD_DELSVAR_TEXT_ID = "fk7263.label.plannedtreatment.other";
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_CATEGORY_ID = "arbetslivsinriktadRehabilitering";
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_CATEGORY_TEXT_ID = "fk7263.label.workrehab";
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_SVAR_ID = "10";
+    public static final String PATIENTENS_ARBETFORMAGA_CATEGORY_ID = "patientensArbetsformoga";
+    public static final String PATIENTENS_ARBETFORMAGA_CATEGORY_TEXT_ID = "fk7263.label.patientworkcapacity";
+    public static final String PATIENTENS_ARBETFORMAGA_SVAR_ID = "11";
+    public static final String PATIENTENS_ARBETFORMAGA_NUVARANDE_ARBETE_TEXT_ID = "fk7263.label.patientworkcapacity.currentwork";
+    public static final String PATIENTENS_ARBETFORMAGA_FORALDRALEDIGHET_TEXT_ID = "fk7263.label.patientworkcapacity.parentalleave";
+    public static final String PATIENTENS_ARBETFORMAGA_ARBETSLOSHET_TEXT_ID = "fk7263.label.patientworkcapacity.unemployed";
```

Comment:
Remaining ARBETFORMAGA in the above constants

Reply:
Fixed.